src/views/DatasetManager/components/DatasetList/DatasetList.js

```diff
@@ -149,10 +140,10 @@ export const DatasetList = () => {
       <Card variant="outlined" square={true} sx={{ backgroundColor: 'transparent', mt: 1 }}>
         <List subheader={datasetListHeader}>
           <Divider />
-          {displayedDatasetList.map((dataset) => (
+          {displayedDatasetList.map((dataset, index) => (
             <ListItemButton
               key={dataset.id}
-              selected={dataset.id === currentDataset?.id}
+              selected={currentDataset?.id ? dataset.id === currentDataset.id : index === 0}
```

Comment:
This change could lead to the dataset list showing that a dataset is selected although no scenario is selected in the "current dataset" redux reducer (meaning that the dataset overview and dataset metadata panels will be empty).

Can we keep the previous condition `dataset.id === currentDataset?.id` here or does it create problems?

The previous version of the code was selecting the default dataset in the DatasetList hooks (`if (currentDataset == null) selectDataset(sortedDatasetList?.[0] ?? null);`), we could restore this line, or move it to another module if you think it is in the wrong place (for instance, we could move this logic in the saga that retrieves all datasets)